doc/advanced.rst

```diff
@@ -180,7 +180,9 @@ The function should take the following inputs (in this order):
    and the image file name is ``'sphx_glr_'`` followed by the name of the
    source ``.py`` file then a number, which starts at 1 and increases by 1 at
    each iteration. The default file format is ``.'png'``. For example:
-   ``'home/user/Documents/module/auto_examples/images/sphx_glr_plot_mymodule_001.png'``
+   ``'home/user/Documents/module/auto_examples/images/sphx_glr_plot_mymodule_001.png'``.
+   The scraper is responsible for replacing the `.png` extension with a supported
+   image extension for it to then be picked up by Sphinx-Gallery.
```

Comment:
Maybe more info:
```
If an extension other than the default ``.png`` is desired, the scraper
is responsible for changing the extension to a supported image
extension, so that it can be picked up by Sphinx-Gallery. Currently,
supported image extensions are: 'png', 'svg', 'jpg', 'gif'.
```